sdk/proguard-rules.pro

```diff
@@ -18,4 +18,11 @@
 
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
-#-renamesourcefileattribute SourceFile
\ No newline at end of file
+#-renamesourcefileattribute SourceFile
+
+-keep class com.cardinalcommerce.dependencies.internal.bouncycastle.**
+-keep class com.cardinalcommerce.dependencies.internal.nimbusds.**
+-keep class tech.dojo.pay.sdk.card.data.entities.**
+-keep class tech.dojo.pay.sdk.card.entities.**
+-keep class tech.dojo.pay.sdk.payemntintent.data.entities.**
+-keep class tech.dojo.pay.sdk.paymentmethods.data.entitiesx.**
```

Comment:
I think this is misspelled. Did it worked using entities I supposed, right?

Reply:
yup it works  will fix that now